package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "alexa-verifier-middleware",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
Looks like we had released 2.0.2 before, so this should be 2.0.3, the next developer iteration.

Reply:
Had changes this to 2.0.3

Reply:
Should I need to update package.json and changeLog too?

Reply:
Probably.

Reply:
Done.